indexers/components/components.sql

```diff
@@ -1,14 +1,22 @@
 CREATE TABLE
   "versions" (
-    "id" SERIAL PRIMARY KEY,
-    "block_height" INT NOT NULL,
+    "id" SERIAL,
+    "block_height" BIGINT NOT NULL,
```

Comment:
why the change to bigint?

Reply:
This change is optional as it will take a lot of time before we hit the max of INT. However, I figured we may do that now since the schema changes anyway which will result in a new indexer. I may revert it if we stick to INT values only.